website/content/docs/platform/k8s/csi/index.mdx

```diff
@@ -10,6 +10,8 @@ description: >-
 The Vault CSI Provider allows pods to consume Vault secrets using
 [CSI Secrets Store](https://github.com/kubernetes-sigs/secrets-store-csi-driver) volumes.
 
+The Vault CSI provider is a fully supported component of Hashicorp Vault.
```

Comment:
We should be spelling this HashiCorp with a capital c (same for the others)

Reply:
Good catch! Updated here: https://github.com/hashicorp/vault/pull/30173/commits/9024329c3cb9a9736dac0650ebd3ae0f236a65b0